website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

```diff
@@ -26,21 +26,19 @@ To upgrade an environment in the [dbt Cloud Admin API](/docs/dbt-cloud-apis/admi
 
 ### Override dbt version
 
-Configure your project to use a different dbt Core version than what's configured in your [development environment](/docs/dbt-cloud-environments#types-of-environments). This _override_ only affects your user account, no one else's. Use this to safely test new dbt features before upgrading the dbt version for your projects. 
+Configure your project to use a different dbt version than what's configured in your [development environment](/docs/dbt-cloud-environments#types-of-environments). This _override_ only affects your user account, no one else's. Use this to safely test new dbt features before upgrading the dbt version for your projects. 
 
 1. Click your account name from the left side panel and select **Account settings**. 
-1. Choose **Credentials** from the sidebar and select a project. This opens a side panel.
-1. In the side panel, click **Edit** and scroll to the **User development settings** section. Choose a version from the **dbt version** dropdown and click **Save**.
+2. Choose **Credentials** from the sidebar and select a project. This opens a side panel.
+3. In the side panel, click **Edit** and scroll to the **User development settings** section. 
+4. Choose a version from the **dbt version** dropdown and click **Save**.
 
-  An example of overriding the configured version with 1.7 for the selected project:
+  An example of overriding the configured version to ["Latest" release track](/docs/dbt-versions/cloud-release-tracks) for the selected project:
 
   <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-override-version.png" width="60%" title="Example of overriding the dbt version on your user account"/>
 
-1. (Optional) Verify that dbt Cloud will use your override setting to build the project. Invoke `dbt build` in the IDE's command bar. Expand the **System Logs** section and find the output's first line. It should begin with `Running with dbt=` and list the version dbt Cloud is using.
-
-  Example output of a successful `dbt build` run: 
-
-  <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/choosing-dbt-version/example-verify-overridden-version.png" title="Example output showing version 1.7 being used, not 1.5"/>
+5. (Optional) Verify that dbt Cloud will use your override setting to build the project by invoking a `dbt build` command in the dbt Cloud IDE's command bar. Expand the **System Logs** section and find the output's first line. It should begin with `Running with dbt=` and list the version dbt Cloud is using. <br /><br />
+   For users on Release tracks, the output will display `Running dbt...` instead of a specific version like `Running dbt=1.9.X`, reflecting the flexibility and continuous updates provided by the release track functionality.
```

Comment:
```suggestion
   For users on Release tracks, the output will display `Running dbt...` instead of a specific version, reflecting the flexibility and continuous automatic updates provided by the release track functionality.
```